new-app/src/App/runGame/RandNum.tsx

```diff
@@ -1,34 +1,42 @@
-import React, {useState} from 'react';
-import * as Code from '../../model/code';
+import React, { useState } from "react";
+import * as Code from "../../model/code";
 
 const RandomNumber = () => {
-  const [randString, setRandString] = useState('')
+  const [randString, setRandString] = useState("");
 
   const getRandomDigit = (): Code.CodeDigit => {
-    const dig = Code.allDigits
-    const rand = Math.floor((Math.random() * dig.length)) as Code.CodeDigit 
-    if (dig.includes(rand)){
-      return rand 
-    } 
-    else {
-      throw new Error(`Failed to get random digit`)
+    const dig = Code.allDigits;
+    const rand = Math.floor(Math.random() * dig.length) as Code.CodeDigit;
+    if (dig.includes(rand)) {
+      return rand;
+    } else {
+      throw new Error(`Failed to get random digit`);
```

Comment:
You don't need to string interpolate this necessarily, since there are no variables in the string. Could just be:

`throw new Error("foo")`